adapter/src/main/java/com/twilio/kudu/sql/schema/KuduSchema.java

```diff
@@ -118,6 +111,29 @@ public void clearCachedTableMap() {
     cachedTableMap = Optional.empty();
   }
 
+  private Pair<String, List<String>> getTableMetadataFromComments(String tableName) {
+    String timeStampColumnName = null;
+    List<String> descendingOrderedColumns = new ArrayList<>();
+    try {
+      KuduTable kuduTable = this.client.openTable(tableName).join();
+      for (ColumnSchema columnSchema : kuduTable.getSchema().getColumns()) {
+        String comment = columnSchema.getComment();
+        if (!comment.isEmpty()) {
+          JSONObject jsonObject = new JSONObject(comment);
+          if (jsonObject.has("isTimeStampColumn") && jsonObject.getBoolean("isTimeStampColumn")) {
+            timeStampColumnName = columnSchema.getName();
+          }
+          if (jsonObject.has("isDescendingSortOrder") && jsonObject.getBoolean("isDescendingSortOrder")) {
```

Comment:
would we need this check only if it is a `isTimeStampColumn` is true?

Reply:
There can be more than one descending column, but there can be only one timestamp column for eg
```
ORDER BY STRING_COL, FLOOR(TIMESTAMP_COLUMN2 TO hour) DESC, INT32_COLUMN2 DESC
```